tests/unit/scripts/test_kg_build.py

```
@@ -151,23 +176,48 @@ async def __aenter__(self) -> "FakeDriver":
+        Simulate executing a Cypher query against the fake Neo4j driver and return a synthetic result set.
+        
+        Parameters:
+            query (str): The Cypher query string to evaluate; matching substrings determine which synthetic response to return.
+            parameters (dict | None): Optional query parameters (unused by response logic but accepted for signature compatibility).
+        
+        Returns:
+            tuple: A 3-tuple (rows, summary, statistics) where `rows` is a list of dictionaries representing query results (often with a `"value"` key for counts), and `summary` and `statistics` are `None` placeholders.
+        """
         query_text = query.strip()
         self.queries.append(query_text)
+        params = parameters or kwargs.get("parameters") or {}
```

Ruff (0.13.2)

Local variable `params` is assigned to but never used

Remove assignment to unused variable `params`

(F841)